epochStart/bootstrap/process.go

```diff
@@ -462,16 +545,21 @@ func (e *epochStartBootstrap) requestAndProcessForShard() error {
 	if err != nil {
 		return err
 	}
+	log.Info("start in epoch bootstrap: GetMiniBlocks")
 
-	shardIds := make([]uint32, 0, 2)
-	hashesToRequest := make([][]byte, 0, 2)
-	hashesToRequest = append(hashesToRequest, epochStartData.LastFinishedMetaBlock)
-	hashesToRequest = append(hashesToRequest, epochStartData.FirstPendingMetaBlock)
-	shardIds = append(shardIds, e.shardCoordinator.SelfId())
-	shardIds = append(shardIds, e.shardCoordinator.SelfId())
+	shardIds := []uint32{
+		core.MetachainShardId,
```

Comment:
Please add a comment why the meta-chain ID twice.

Reply:
the name of the next hashes have the suffix META